File: cs/f00.inheritence/Student.cs
public class Student : Person
{
    private double gpa;

    // Constructors
    //public Student(string name) { this.Name = name; }

    // Properties
    public double GPA {
        get { return gpa; }    // accessor - get data
        set { gpa = value; }   // mutator  - set data
    }
}

File: python/f00.inheritence/Student.py
#!/usr/bin/python
from Person import Person    # include Person.py

# class declaration
class Student(Person):
  ###### Initializer ######
  #def __init__(self,gpa):
  #  self.setGPA(gpa)    # call mutator

  ###### Mutators ######
  def setGPA (self,gpa):
    self.gpa = gpa      # set data

  ###### Accessors ######
  def getGPA(self):
    return self.gpa      # return data
